app/Http/Controllers/api/v1/RoomController.php

```diff
@@ -119,31 +119,74 @@ public function start(Room $room, StartJoinMeeting $request)
 
         $meeting = $room->runningMeeting();
         if (!$meeting) {
-            $servers = Server::where('status', true)->get();
-
-            try {
-                $server = $servers->random();
-            } catch (InvalidArgumentException $ex) {
-                abort(CustomStatusCodes::NO_SERVER_AVAILABLE, __('app.errors.no_server_available'));
-            }
-
-            $meeting = $room->meetings()->create();
-            $meeting->server()->associate($server);
+            // Create new meeting
+            $meeting              = $room->meetings()->create();
             $meeting->start       = date('Y-m-d H:i:s');
             $meeting->attendeePW  = bin2hex(random_bytes(5));
             $meeting->moderatorPW = bin2hex(random_bytes(5));
             $meeting->save();
 
-            if (!$meeting->startMeeting()) {
-                abort(CustomStatusCodes::ROOM_START_FAILED, __('app.errors.room_start'));
+            // Try to find server to host meeting
+            while (true) {
```

Comment:
The loop is not the best solution! There are some cases where the meeting gets deleted and the loop runs a second time. In this cases an error will be thrown on line 144.